### ISSUE TITLE
python installation for pyabcranger 0.0.53 broken

### ISSUE THREAD

**Issue by psteinb** [diyabc/abcranger#93]
Just wanted to document this, the pypi release of `pyabcranger` for version 0.0.53 is broken. My local system requires me to to build the package locally:

```
$ python -m pip install pyabcranger==0.0.53
Collecting pyabcranger==0.0.53
  Using cached pyabcranger-0.0.53.tar.gz (51 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-8e3mi72v/pyabcranger_fff78bf30d8947349c8e5d083a7e2c0a/setup.py", line 1, in <module>
          from cmaketools import setup
      ModuleNotFoundError: No module named 'cmaketools'
      [end of output]
#...
```

Once installing `cmaketools` and `wheel`, I get this:
```
$ python -m pip install pyabcranger==0.0.53
Collecting pyabcranger==0.0.53
  Using cached pyabcranger-0.0.53.tar.gz (51 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [109 lines of output]
      PYTHON VERSION : 3.9
      running egg_info
      egg_info (cmake)
      creating /tmp/pip-pip-egg-info-k6e2tcp3/pyabcranger.egg-info
      writing /tmp/pip-pip-egg-info-k6e2tcp3/pyabcranger.egg-info/PKG-INFO
      writing dependency_links to /tmp/pip-pip-egg-info-k6e2tcp3/pyabcranger.egg-info/dependency_links.txt
      writing top-level names to /tmp/pip-pip-egg-info-k6e2tcp3/pyabcranger.egg-info/top_level.txt
      writing manifest file '/tmp/pip-pip-egg-info-k6e2tcp3/pyabcranger.egg-info/SOURCES.txt'
      Cloning into 'ranger'...
      Cloning into 'wingetopt'...
      -- The CXX compiler identification is GNU 11.3.1
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /usr/bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      Compiler ID : GNU
      -- The C compiler identification is GNU 11.3.1
#...
-- Found BLAS: -Wl,--start-group /opt/intel/mkl/lib/intel64/libmkl_intel_lp64.a /opt/intel/mkl/lib/intel64/libmkl_tbb_thread.a /opt/intel/mkl/lib/intel64/libmkl_core.a -Wl,--end-group;pthread;m;dl
      -- Found LAPACK: -Wl,--start-group /opt/intel/mkl/lib/intel64/libmkl_intel_lp64.a /opt/intel/mkl/lib/intel64/libmkl_tbb_thread.a /opt/intel/mkl/lib/intel64/libmkl_core.a -Wl,--end-group;pthread;m;dl
      -- [pmm] Using vcpkg executable: /home/steinb95/.local/share/pmm/1.5.1/vcpkg-35c98ef06265660f036a3156d7615c735a10087c/vcpkg
      CMake Error at build/_pmm/1.5.1/vcpkg.cmake:154 (message):
        Failed find portfile in
        /tmp/pip-install-xhm3duu3/pyabcranger_4d6adccd776c4635a4a03a22440e1534/pybind11-nodep!
      Call Stack (most recent call first):
        build/_pmm/1.5.1/vcpkg.cmake:202 (_pmm_vcpkg_copy_custom_ports)
        build/_pmm/1.5.1/main.cmake:38 (_pmm_vcpkg)
        build/_pmm/1.5.1/main.cmake:52 (_pmm_project_fn)
        CMakeLists.txt:86 (pmm)
      
      
      -- Configuring incomplete, errors occurred!
      See also "/tmp/pip-install-xhm3duu3/pyabcranger_4d6adccd776c4635a4a03a22440e1534/build/CMakeFiles/CMakeOutput.log".
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-xhm3duu3/pyabcranger_4d6adccd776c4635a4a03a22440e1534/setup.py", line 23, in <module>
          setup(
        File "/tmp/venv-py39/lib/python3.9/site-packages/cmaketools/__init__.py", line 98, in setup
          _setup(**setup_args)
        File "/tmp/venv-py39/lib/python3.9/site-packages/setuptools/__init__.py", line 153, in setup
          return distutils.core.setup(**attrs)
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/core.py", line 148, in setup
          dist.run_commands()
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/dist.py", line 966, in run_commands
          self.run_command(cmd)
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/dist.py", line 985, in run_command
          cmd_obj.run()
        File "/tmp/venv-py39/lib/python3.9/site-packages/setuptools/command/egg_info.py", line 299, in run
          self.find_sources()
        File "/tmp/venv-py39/lib/python3.9/site-packages/cmaketools/cmakecommands.py", line 72, in find_sources
          mm.run()
        File "/tmp/venv-py39/lib/python3.9/site-packages/setuptools/command/egg_info.py", line 541, in run
          self.add_defaults()
        File "/tmp/venv-py39/lib/python3.9/site-packages/setuptools/command/egg_info.py", line 578, in add_defaults
          sdist.add_defaults(self)
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/command/sdist.py", line 228, in add_defaults
          self._add_defaults_ext()
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/command/sdist.py", line 311, in _add_defaults_ext
          build_ext = self.get_finalized_command('build_ext')
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/cmd.py", line 299, in get_finalized_command
          cmd_obj.ensure_finalized()
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/distutils/cmd.py", line 107, in ensure_finalized
          self.finalize_options()
        File "/tmp/venv-py39/lib/python3.9/site-packages/cmaketools/cmakecommands.py", line 452, in finalize_options
          self.cmake.configure(self.build_base, **cmake_settings)
        File "/tmp/venv-py39/lib/python3.9/site-packages/cmaketools/cmakebuilder.py", line 321, in configure
          cmakeutil.configure(".", build_dir, *args, **kwargs)
        File "/tmp/venv-py39/lib/python3.9/site-packages/cmaketools/cmakeutil.py", line 120, in configure
          sp.run(args, env=env, check=True).check_returncode()
        File "/home/steinb95/software/python/3.9.9/lib/python3.9/subprocess.py", line 528, in run
          raise CalledProcessError(retcode, process.args,
      subprocess.CalledProcessError: Command '['cmake', '.', '-B', 'build', '-DPYTHON_EXECUTABLE=/tmp/venv-py39/bin/python', '-DPYBIND11_PYTHON_VERSION=3.9', '-DPYABCRANGER=TRUE', '-DUSE_MKL:BOOL=TRUE', '-DMAKE_STATIC_EXE:BOOL=TRUE', '-DLAPACK_ROOT:STRING=/opt/intel/mkl/lib/intel64', '-DLAPACK_LIBRARIES:STRING=-Wl,--start-group /opt/intel/mkl/lib/intel64/libmkl_intel_lp64.a /opt/intel/mkl/lib/intel64/libmkl_tbb_thread.a /opt/intel/mkl/lib/intel64/libmkl_core.a -Wl,--end-group;pthread;m;dl', '-DBLAS_LIBRARIES:STRING=-Wl,--start-group /opt/intel/mkl/lib/intel64/libmkl_intel_lp64.a /opt/intel/mkl/lib/intel64/libmkl_tbb_thread.a /opt/intel/mkl/lib/intel64/libmkl_core.a -Wl,--end-group;pthread;m;dl', '-G', 'Ninja', '-DCMAKE_BUILD_TYPE:STRING=Release']' returned non-zero exit status 1.
      [git] .gitmodules found. Cloning the submodules if necessary
      [git] cloning https://github.com/fradav/ranger.git to ranger...
      [git] cloning complete.
      [git] cloning https://github.com/alex85k/wingetopt.git to wingetopt...
      [git] cloning complete.
      
      [cmake] configuring CMake project...
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by fradav** [comment]
Hi, thanks for your interest for the software. This is basically due to the same issue than #92, lack of python 3.10 wheel some updates with the github CI banzaied the whole PyPi wheel generation. I’m looking into it.

**Comment by psteinb** [comment]
Good to know, that you are aware. :heavy_check_mark:  Thanks also for providing this software. This is greatly appreciated. :raised_hands: 

This issue is mostly for documentation, so that other users are informed too.

**Comment by fradav** [comment]
Fixed as #92
